dotnet/samples/KernelSyntaxExamples/Example76_TextEmbedding.cs

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using Microsoft.SemanticKernel.Text;
+using RepoUtils;
+using SharpToken;
+
+public static class Example76_TextEmbedding
+{
+    private const string EmbeddingModelName = "text-embedding-ada-002";
+    private const string ChatTranscript =
```

Comment:
nit: It would be nice to hide this chat transcript somewhere lower in this file under region or maybe put it in separate txt file, so when developers open an example, they see actual code. I think the content of chat transcript doesn't matter in this example, please let me know if I'm missing something.

Reply:
Moved transcript to the bottom of the file and inlined the embedding model name for clarity.